_data/settings.yml

```diff
@@ -1,3 +1,8 @@
 talk:
-    isLive: False
-    speaker: Hannah Kerner
+  isLive: False
+  speaker: Hannah Kerner
+
+donations:
+  spreadsheetURL: https://docs.google.com/spreadsheets/d/e/2PACX-1vTXKpZ8iB5Nz6mHl5iYUAA68DU1D-gCnwuDREnVULI9JrOnh1CF8U1RF6zDbVfrkw/pub?gid=1443797343&single=true&output=csv
+  target: 100000
+  targetTextForm: "100,000.00"
```

Comment:
I would remove the floating part, to simplify the reading.

Reply:
You mean you would write it as 100 000 such that it appears as "100 000 CAD" in the text?